OCP monitoring 4.17 release notes (#82009)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

[id="ocp-4-17-monitoring_{context}"]
=== Monitoring

The in-cluster monitoring stack for this release includes the following new and modified features. 
//change the line if no features

[id="ocp-4-17-monitoring-updates-to-monitoring-stack-components-and-dependencies"]

> ==== Updates to monitoring stack components and dependencies
> This release includes the following version updates for in-cluster monitoring stack components and dependencies:

Reviewer: ```suggestion
==== Updates to monitoring stack components and dependencies

This release includes the following version updates for in-cluster monitoring stack components and dependencies:
```

* Alertmanager to 0.27.0
* Prometheus Operator to 0.75.2
* Prometheus to 2.53.1
* kube-state-metrics to 2.13.0
* node-exporter to 1.8.2
* Thanos to 0.35.1

[id="ocp-4-17-monitoring-changes-to-alerting-rules"]
==== Changes to alerting rules

[NOTE]
====
Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
====

* Added the `PrometheusKubernetesListWatchFailures` alert to warn users about Prometheus and Kubernetes API failures, such as unreachable API and permissions issues, which can lead into silent service discovery failures.

[id="ocp-4-17-monitoring-prometheus-updated-to-tolerate-jitters-at-scrape-time-uwm"]

> ==== Prometheus updated to tolerate jitters at scrape time for user-defined projects

Reviewer: ```suggestion
==== Updated Prometheus to tolerate jitters at scrape time for user-defined projects
```

> The Prometheus configuration for monitoring for user-defined projects has been updated so that jitters are tolerated at scrape time.

Reviewer: ```suggestion
The Prometheus configuration for monitoring for user-defined projects is updated to tolerate jitters at scrape time.
```

> For monitoring deployments that have shown sub-optimal chunk compression for data storage, this update helps to optimize data compression, thereby reducing the disk space used by the time series database in these deployments.

Reviewer: To avoid the passive voice:

```suggestion
With this update, you can optimize data compression for monitoring deployments that show sub-optimal chunk compression for data storage. You can also reduce the disk space used by the time series database in monitoring deployments.
```
Please tweak the suggestion if required - by avoiding the passive voice.

Author: Thank you for the suggestion. I will use a part of it. But there should not be "you can" as this is not something the user can do, it is what the update does. So I will tweak it based on that. Thank you!
